test/jdk/java/net/NetworkInterface/UniqueMacAddressesTest.java

```diff
@@ -21,87 +21,65 @@
  * questions.
  */
 
-import java.net.InetAddress;
+import java.io.PrintStream;
+import java.io.UncheckedIOException;
 import java.net.NetworkInterface;
 import java.net.SocketException;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Enumeration;
 import java.util.List;
+import java.util.stream.Collectors;
 
+import jdk.test.lib.NetworkConfiguration;
 
 /*
  * @test
  * @bug 8021372
- * @summary Tests that the MAC addresses returned by NetworkInterface.getNetworkInterfaces are unique for each adapter.
- *
+ * @summary Tests that the MAC addresses returned by NetworkConfiguration.probe() are unique for each adapter.
```

Comment:
I would keep the summary as in the original test. We are really testing the NetworkInterfaces returned by `NetworkInterface.getNetworkInterfaces` - even though we are skipping some of them that we know are problematic.